odxtools/service.py

```diff
@@ -27,6 +27,7 @@ def __init__(self,
                  long_name: Optional[str] = None,
                  description: Optional[str] = None,
                  semantic: Optional[str] = None,
+                 addressing: Optional[str] = None,
```

Comment:
Do we still need the Optional, since we are defaulting?

Reply:
Hmm, yeah that's true. Probably not haha